Assets/HoloToolkit/SpatialUnderstanding/Scripts/SpatialUnderstandingCustomMesh.cs

```diff
@@ -98,6 +98,11 @@ private class MeshData
             public readonly Mesh MeshObject = new Mesh();
 
             /// <summary>
+            /// The MeshCollider with which this mesh is associated.
+            /// </summary>
+            public MeshCollider Collider = null;
```

Comment:
I suggest using a more specific name. Collider is a unity class.  I suggest SpatialCollider or something along those lines.